pkgs/by-name/ot/otb/package.nix

```diff
@@ -0,0 +1,162 @@
+{ cmake
+, callPackage
+, fetchFromGitHub
+, makeWrapper
+, lib
+, stdenv
+, swig
+, which
+, boost
+, curl
+, gdal
+, libsvm
+, libgeotiff
+, muparser
+, muparserx
+, opencv
+, perl
+, python3
+, shark
+, tinyxml
+, enableFeatureExtraction ? true
+, enableHyperspectral ? true
+, enableLearning ? true
+, enableMiscellaneous ? true
+, enableOpenMP ? false
+, enablePython ? true
+, extraPythonPackages ? ps: with ps; [ ]
+, enableRemote ? true
+, enableSAR ? true
+, enableSegmentation ? true
+, enableStereoProcessing ? true
```

Comment:
@SuperSandro2000  I have made some changes just to allow for optional arguments for each module in OTB.